packages/renderless/src/numeric/vue.ts

```diff
@@ -139,16 +139,23 @@ const initWatch = ({
   state,
   watch,
   props,
-  api
+  api,
 }: Pick<INumericRenderlessParams, 'state' | 'watch' | 'props' | 'api'>): void => {
+  
+  watch(() => [props.max, props.min], ([curMax,curMin]) => {
+    if (curMax = curMin){
+      throw new Error('[Numeric]: The maximum value should not be less than to the minimum value');
+    }
+  }, {immediate: true})
```

Comment:
@GaoNeng-wWw 
主要有以下问题：
1. 语句末尾不应该加分号`;`
2. 多处缺失必要的空格
3. `=` 应该是 `<`，这里估计是手误吧

```suggestion
  watch(() => [props.max, props.min], ([curMax, curMin]) => {
    if (curMax < curMin) {
      throw new Error('[Numeric]: The maximum value should not be less than to the minimum value')
    }
  }, { immediate: true })
```

Reply:
Fixed